assets/styles/main.scss

```diff
@@ -54,3 +54,6 @@
 // Import city specific style overrides.
 
 @import "~open-city-design/src/scss/helsinki/custom-styles";
+//@import "~react-image-crop/lib/ReactCrop";
+
+@import "../js/components/Tab/Profile/style.scss"
```

Comment:
The changes to this file can be discarded. I don't see a file called `style.scss` in Profile component directory.